src/js/components/DataTable/Body.js

```diff
@@ -182,7 +184,9 @@ const Body = forwardRef(
               }
             : undefined
         }
+        // TODO: skip disabled
```

Comment:
Should we create an issue to change the arrow key behavior for disabled options to skip them?